### DISCUSSION TITLE
LocationModelLinearDependentW model in the new cell2location version

### DISCUSSION THREAD

**Discussion by sokratiag** [BayraktarLab/cell2location#328] [General]
Hi,

I am wondering if it is still possible to use the 'old' LocationModelLinearDependentW with the new cell2location version? 

I used cell2location with one of the very first versions (0.04alpha) but now I am not able to reproduce the results with the latest c2l version. 

Thanks!

**Comment by sokratiag** [comment]
I think I managed to do this with the LocationModelLinearDependentWMultiExperimentLocationBackgroundNormLevelNoMGPyroModel class. 

Thanks.

**Comment by vitkl** [comment]
What kind of result are you trying to reproduce?

I would not recommend using `LocationModelLinearDependentWMultiExperimentLocationBackgroundNormLevelNoMGPyroModel` because it lacks `m_g` variable that accounts for the difference in RNA detection sensitivity across genes between single cell reference and spatial data. The previous (pymc, 0.04alpha) version of cell2location also had numerical accuracy issues.

**Comment by sokratiag** [comment]
Thanks for your reply vitkl. I am working with liver samples, and with the new version I am not able to get a nice mapping of a subtype of hepatocytes which we expect those types to be on a particular niche but rather I get a more global /dense mapping of this subtype. With the no m_g model I am however able to map that  to their niche. I know that in your preleased version (0.04a) the m_g term was included but the priors look a bit different with the new version. I am not sure whether the priors could cause this behaviour? Any recommendations? Thanks!
